skopt/searchcv.py

```diff
@@ -675,6 +690,29 @@ def step(self, X, y, space_id, groups=None, n_jobs=1):
         if self.refit:
             self._fit_best_model(X, y)
 
+    def total_iterations(self):
```

Comment:
Should we make this a property of `skopt.Space`? Somehow feels like that is where this should be. Need a good solution to adding up sub-spaces though.

Reply:
Hm
What this does is it simply sums over all iterations that are set for the subspaces provided to the Optimizer. One can decide on different number of iterations for same subspace depending on how much patience and resources one has to wait for the algorithm to terminate. So far this seems rather like a property of optimizer than space. So I am -0.5 on moving this to space, especially if we do not have a way to define subspaces there yet